packages/cursorless-engine/src/generateSpokenForm/primitiveTargetToSpokenForm.ts

```diff
@@ -49,6 +49,17 @@ export class PrimitiveTargetSpokenFormGenerator {
         throw new NoSpokenFormError(`Modifier '${modifier.type}'`);
 
       case "containingScope":
+        if (modifier.ancestorIndex != null) {
+          if (modifier.ancestorIndex === 1) {
```

Comment:
I think `modifier.ancestorIndex === 0` should behave the same as `modifier.ancestorIndex == null`

Reply:
fixed